browser/js/content.js

```diff
@@ -0,0 +1,115 @@
+"use strict";
+
+function init() {
+  // Only do work on reddit.
+  if (!location.hostname.endsWith('.reddit.com')) return;
+
+  // FIXME: This can be more declarative in the future.
+  if (checkCompose.test(location.pathname)) injectCompose();
+  else if (checkProfile.test(location.pathname)) injectProfile();
+  else if (checkThread.test(location.pathname)) injectThread();
+}
+window.addEventListener('load', init);
+
+
+const checkCompose = /^\/message\/compose$/
+function injectCompose() {
+  // /message/compose
+  // TODO: ...
+  console.log("keybase: On compose.");
+}
+
+const checkProfile = /^\/user\//;
+function injectProfile() {
+  // /user/<user>
+  // TODO: ...
+  console.log("keybase: On profile.");
+}
+
+const checkThread = /^\/r\/\w+\/comments\/\w+\//;
+function injectThread() {
+  // /r/<subreddit>/comments/<id>/<slug>
+  console.log("keybase: On thread.");
+
+  for (let c of document.getElementsByClassName("comment")) {
+    const author = escape(c.getAttribute("data-author"));
```

Comment:
I think the correct thing to do here is actually `foo.replace(/</g, "&lt;").replace(/>/g, "&gt;")` (or use a templating library). I'm planning on having a fix for the next PR but I wasn't considering it very high priority because:
1. usernames are alphanumeric, so the input shouldn't be a problem to begin with
2. `escape` is not going anywhere anytime soon (despite being deprecated for forever)
3. encodeURIComponent and friends are still not technically correct for general use because they escape way too many things (focused for URI encoding, not for safe html rendering).